src/main/webapp/styles/personalised-recommendations.css

```diff
@@ -0,0 +1,91 @@
+#instructions {
+  color: black;
+}
+
+/* Wrapper Containing Preference Form and Recommendations */
+.wrapper {
+  display: flex;
+  width: 50%;
+}
+
+.preferences-container {
```

Comment:
can we make the  preferences-container to align center

Reply:
I plan to display the recommendation result on the right, as discussed in the wireframe in the design doc.